SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -61,12 +61,23 @@ struct FConnectionConfig
 struct FReceptionistConfig : public FConnectionConfig
 {
 	FReceptionistConfig()
-		: ReceptionistHost(SpatialConstants::LOCAL_HOST)
-		, ReceptionistPort(SpatialConstants::DEFAULT_PORT)
+		: ReceptionistPort(SpatialConstants::DEFAULT_PORT)
 	{
 		const TCHAR* CommandLine = FCommandLine::Get();
 
-		FParse::Value(CommandLine, TEXT("receptionistHost"), ReceptionistHost);
+		// Parse the commandline for receptionistHost, if it exists then use this as the host IP.
+		if (!FParse::Value(CommandLine, TEXT("receptionistHost"), ReceptionistHost))
+		{
+			// If receptionistHost is not specified then parse for a traditional IP.
+			if (!FParse::Token(CommandLine, ReceptionistHost, 0) || **ReceptionistHost == '-' || **ReceptionistHost == '/')
+			{
+				// If an IP is not specified then use default.
+				UE_LOG(LogTemp, Warning, TEXT("Could not find 'receptionistHost' or IP address in arguments for ReceptionistConfig. Using default: %s"), *SpatialConstants::LOCAL_HOST);
```

Comment:
Please don't use LogTemp. Either create a category, or absorb this log into another suitable one.